src/app/components/home-page/home-page.component.css

```diff
@@ -0,0 +1,89 @@
+body {
+	font-family: Helvetica, Arial, sans-serif;
+}
+
+.home {
+	position: relative;
+	display: block;
+}
+
+h1 {
+	color: #2F6A75;
+}
+
+button {
+	color: white;
+	background-color: #01D1D9;
+	padding: 10px 16px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 13px;
+    margin: 4px 2px;
+    cursor: pointer;
+    border: none;
+    border-radius: 5px;
+    outline: none;
+    position: relative;
+    left: 562px;
+} 
+
+.container {
+	background-color: white;
+	width: 715px;
+	height: 320px;
+	border-radius: 5px;
+}
+
+#day {
```

Comment:
try not to have many ids. use mostly classes. advantages of ids is to use them to get dom element quickly, but at the same time is not reusable.